docs/conf.py

```diff
@@ -81,3 +91,8 @@
     "colon_fence",
     "deflist",
 ]
+
+# One entry per manual page. List of tuples
+# (source start file, name, description, authors, manual section)
+
+man_pages = [("man", "scikit-build-core", f"{project} {release}", [author], 1)]
```

Comment:
Shouldn't this page be excluded from `html`? Just so it won't create warnings and be indexed/navigable manually.

Reply:
Yes, it's not listed in index.md & has an `isolated` marker to avoid the warning.

Reply:
I guess you mean the `orphan` directive. I think with that the page is still manually navigable. Dunno about search-indexing though. Why not enable rtd for PRs to check these?

Reply:
Oops, yes, that's what I meant. :)

Hmm, not too worried about it being generated (don't see an easy way to stop it - the "only" directive doesn't work for changing the structure), but showing up in search is a bit annoying. I'm pushing a fix for that, though.